service/src/main/java/uk/gov/hmcts/reform/fpl/service/LegalRepresentativeService.java

```diff
@@ -20,6 +24,14 @@
     private final OrganisationService organisationService;
     private final LegalRepresentativesDifferenceCalculator differenceCalculator;
 
+    public List<Element<LegalRepresentative>> getDefaultLegalRepresentatives(CaseData caseData) {
+        if (ObjectUtils.isEmpty(caseData.getLegalRepresentatives())) {
+            return wrapElements(LegalRepresentative.builder().build());
```

Comment:
I had completely forgotten about this method, I think that it needs changing as it doesn't set the id of the element. I am not sure how ccd reacts to a null id being passed. The same for the list implementation as well.